packages/client/src/main.scss

```diff
@@ -0,0 +1,8 @@
+@import "components/HelloWorld.scss";
```

Comment:
There are a lot of ways to manage scss files. This is just one proposal. Often breaking these imports up makes sense. Something like `components/all.scss` which contains all the component file imports, and then `main.scss` imports `components/all.scss`. Also, it may be possible to get glob imports working with parcel.